asset/src/kafka_reader/slicer.ts

```diff
@@ -6,6 +6,10 @@ export default class KafkaSlicer extends Slicer<KafkaReaderConfig> {
         return Boolean(this.executionConfig.autorecover);
     }
 
+    isRestartable(): boolean {
```

Comment:
```suggestion
    isRelocatable(): boolean {
```

We've decided to switch to `isRelocatable`.  This should be changed here and in the appropriate places in Teraslice.  We will write an issue to consider adding a `relocating` state.